packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -175,10 +185,55 @@ export interface TaskDefinitionProps extends CommonTaskDefinitionProps {
   readonly pidMode?: PidMode;
 }
 
+/**
+ * The common task definition attributes used across all types of task definitions.
+ */
+export interface CommonTaskDefinitionAttributes {
+  /**
+   * The arn of the task definition
+   */
+  readonly taskDefinitionArn: string;
+
+  /**
+   * The networking mode to use for the containers in the task.
+   *
+   * @default NetworkMode.BRIDGE
+   */
+  readonly networkMode?: NetworkMode;
+
+  /**
+   * The name of the IAM role that grants containers in the task permission to call AWS APIs on your behalf.
+   *
+   * @default undefined.
```

Comment:
While `@default undefined` here is technically true, it's not very useful.  Anybody who knows JavaScript/TypeScript will know that if you don't pass a property somewhere, it will count as `undefined`.

What is more interesting is to write what the BEHAVIOR will be if you leave the value out. In this case something like `@default - Permissions cannot be granted to the imported task`.

Reply:
Will do.